framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/database/driver/Db2ExternalDriver.java

```diff
@@ -18,32 +18,17 @@
 public class Db2ExternalDriver extends AbstractExternalDriver {
 
     @Override
-    public String getImageName() {
-        return "db2-driver-image";
+    public String getName() {
+        return "db2";
     }
 
     @Override
-    public String getImageVersion() {
-        return "10.5";
-    }
-
-    @Override
-    public String getDockerFileRelativePath() {
-        return "db2-driver-image";
-    }
-
-    @Override
-    public String getCustomInstallDirectories() {
-        return "db2-driver/extensions";
-    }
-
-    @Override
-    public String getSourceImagePath() {
-        return "/extensions:db2-driver/";
+    public String getImageName() {
+        return "jboss-kie-db2-extension-openshift-image";
     }
 
     @Override
-    protected String getDriverBinaryFileLocationInArtifactRepo() {
-        return "com/ibm/db2/jcc/db2jcc4/10.5/db2jcc4-10.5.jar";
+    public String getImageVersion() {
```

Comment:
Well, the constants are used just in one place. I don't expect them to be used on other places.
Adding constant class would bring more code, but all the values would be stored in one place.
I don't have a strong opinion about what to choose in this case, what do you think about it?

Reply:
It depends on how we want to admin it :)

The question would be: Do we need to know all the versions of all the databases at once ?
I am not sure about that.
Anyway, to find the correct version, we would dig into the code to find back which version ...

"I don't expect them to be used on other places" that is true

Finally, I think it would not make many improvmeent here ... We can let it like that for now.

Reply:
" Do we need to know all the versions of all the databases at once ?" No.
About digging into the code to find a version - that is true. I am not sure how often we will upgrade drivers. Hopefully not much often.
In case it will become limiting we can refactor the implementation later.